python/rpdk/go/templates/Makefile

```diff
@@ -2,7 +2,7 @@
 
 build:
 	cfn generate
-	env GOOS=linux go build -ldflags="-s -w" -tags="logging callback metrics scheduler" -o bin/handler cmd/main.go
+	env GOOS=linux go build -ldflags="-s -w" -tags="logging callback scheduler" -o bin/handler cmd/main.go
```

Comment:
Really?

Reply:
Yeah, It's because the metrics tag is not need.

Reply:
Do you think we should keep a tag option that toggles the metrics?